mantle/cmd/kolet/kolet.go

```diff
@@ -271,23 +272,27 @@ func runExtUnit(cmd *cobra.Command, args []string) error {
 	// proxy it into a channel
 	rebootChan := make(chan string)
 	errChan := make(chan error)
-	err := exec.Command("mkfifo", rebootRequestFifo).Run()
-	if err != nil {
-		return err
-	}
-	go func() {
-		rebootReader, err := os.Open(rebootRequestFifo)
-		if err != nil {
-			errChan <- err
-			return
-		}
-		defer rebootReader.Close()
-		buf, err := ioutil.ReadAll(rebootReader)
+
+	// We want to prevent certain tests (like non-exclusive tests) from rebooting
+	if !rebootOff {
```

Comment:
How about now enhancing `runReboot` to check for the existence of `rebootRequestFifo` and providing a clear error if it's missing to explain that reboots are not supported?